### ISSUE TITLE
valgrind FTBFS on 10.12+

### ISSUE THREAD

**Issue by dmacks** [fink/fink-distributions#217]
I (and bindist) have a ./configure error on 10.13:

```
checking for a supported version of gcc... Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
no (9.1.0)
configure: error: please use gcc >= 3.0 or clang >= 2.9 or icc >= 13.0
```

According  to the PatchScript, it only allows llvm up through 8.x. Upstream has a newer version and their git has a [change to allow 9.x](https://sourceware.org/git/?p=valgrind.git;a=commitdiff;h=27e1503bc7bd767f3a98824176558beaa5a7c1d5). That gets me further, but then a different error later:

```
checking for a supported version of gcc... Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
ok (Apple LLVM version 9.1.0)
checking build system type... x86_64-apple-darwin17.7.0
checking host system type... x86_64-apple-darwin17.7.0
checking for a supported CPU... ok (x86_64)
checking for a 64-bit only build... no
checking for a 32-bit only build... no
checking for a supported OS... ok (darwin17.7.0)
checking for the kernel version... unsupported (17.7.0)
configure: error: Valgrind works on Darwin 10.x, 11.x, 12.x, 13.x, 14.x and 15.x (Mac OS X 10.6/7/8/9/10/11)
```
There are a series of git changes that add support for 10.12 and preliminary for 10.13. I'm not sure which have actually made it into the current 3.13.0 release (fink is at 3.11.0).

**Comment by danielj7** [comment]
Yeah, there is no released version that works with xcode 9 or 10.13. The git HEAD version should work. I'll look at making a snapshot.

**Comment by danielj7** [comment]
Now fixed in master.
